docs/reference/predicates.md

```diff
@@ -716,6 +716,32 @@ catalog:
     "https://catalog";
 ```
 
+## TrafficSegment
+
+TrafficSegment predicate requires two number arguments _min_ and _max_
+from an interval [0, 1] (from zero included to one included) and _min_ <= _max_.
+
+Let _r_ be one-per-request uniform random number value from [0, 1).
+TrafficSegment matches if _r_ belongs to an interval from [_min_, _max_).
+Upper interval boundary _max_ is excluded to simplify definition of
+adjacent intervals - the upper boundary of the first interval
+then equals lower boundary of the next and so on, e.g. [0, 0.25) and [0.25, 1).
```

Comment:
We can use math quotes similar to admissionControl filter doc: `$[1, 100]$` renders $[1, 100]$.
I think we can use it also for $[min, max)$ for example

Reply:
Fixed